### ISSUE TITLE
problem in OAG-paper dataset splitting

### ISSUE THREAD

**Issue by initzhang** [jadadong/GNS#4]
in the code below, `valid_labal_idx`'s length is actually the number of all nodes coming with labels (train+test+val). So `train_size` should be `int(len(valid_labal_idx) * (0.43/0.53))` instead of `int(len(valid_labal_idx) * 0.43)`. Current code actually yields a splitting with smaller training & validation set while a much bigger test set, namely Train:Val:Test = 0.53\*0.43: 0.53\*0.05 : 0.53\*(1-0.48). 
https://github.com/jadadong/GNS/blob/b5b2ce89449babc2816781119b032a4490538c72/GNS_sampling_prob.py#L501-L508
